### ISSUE TITLE
retire the translation team

### ISSUE THREAD

**Issue by kjaymiller** [BlackPythonDevs/blackpythondevs#143]
Currently we have a team labeled the translation team.

[Translation Team Definition](../roles/translation_team.md)

There has been a lot of desire from the beginning of Black Python Devs.

Unfortunately, we have not been able to get a commitment to maintain the translations of the website with any regularity. We also have not been able to appoint individuals to take ownership of these tasks.

Therefore, it is my recommendation that we dissolve the translation team until we are able to field a team that can maintain the existing website.

## What will we do about translations?

I don't know what to suggest here.

Translations are a moving target. I worry that we won't be able to handle accessibility and translations effectively.

I look at other websites that are much larger than we are. Most of their site is not translated. This includes -

Python.org
Djangoproject.org
PyLadies.com

Usually the parts that are translated are documentation and policy documents.

My suggestion is that we implement a similar approach.

I think we focus translation efforts onto policy and program pages in GitHub while leaving our primary website in English.

**Comment by yokwejuste** [comment]
Thanks for bringing this up @kjaymiller and thanks for your detail analysis and suggestion. I agree that maintaining translations without a dedicated team is very challenging.
- Dissolution of the Translation Team: This seems like a practical step given the current circumstances.
- Focus on Important Documents: Implementing a focused translation approach for policy and program pages on GitHub while keeping the main website in English makes sense.

I believe right now we can proceed on that; we can revisit the website translations in the future when we have more resources.

**Comment by AlphaCodz** [comment]
Yes @kjaymiller, I'm kinda compelled to concur, it's been a busy year, my sincerest apologies for not being able to keep up with the translations.

**Comment by kjaymiller** [comment]
I've shared this with the leadership council as well and will give them 48 hours to comment before taking action (expires 13 Dec 2024)

**Comment by lazouich** [comment]
+1 for retiring the team. With the built in translation capabilities of most browsers the efforts can better be spent elsewhere until a time comes where there is a greater need for human translated pages.

**Comment by sabderemane** [comment]
I was part of the people who has the desire to translate the website and I didn't managed to do it, unfortunately.

I understand the fact of retirement of the translation team, without the appropriate team it's hard to keep translations accurate and contribute regularly.

Nevertheless, I don't think it should be abandoned. Language is also part of having the website accessible to everyone and I know it's in the values of BPD to welcome everyone. Could we instead put a banner to say that this language is incomplete that translators are needed with a link to contribute? It will not relay only on the current translation team, they could review and it will also help folks to contribute. Maybe not the ideal solution but IMO it worth to try before to dissolve the translation team.

The best example to illustrate this is the [Django Girls+ tutorial](https://tutorial.djangogirls.org/). Other languages not fully translated are shown as beta.

Note that for the Django side, there is on going work for translating all the website but [it's a lot of work](https://github.com/django/djangoproject.com/issues/1648).

**Comment by kjaymiller** [comment]
> Could we instead put a banner to say that this language is incomplete that translators are needed with a link to contribute? 

I am open to the idea. The challenge that I have to this is that we have two forms of incomplete. Data that has changed and data that doesn't exist. I think we would need to look at what we're doing now and how we could indicate pages missing translation and pages needing to be updated. 

In order to do this I would ask if the @BlackPythonDevs/translations  Team can help with that. 

The biggest challenge is that this site being Jekyll will likely need some processing during the build to ensure accuracy.
